Ticket-booking-spring-app/src/main/java/com/example/demo/service/room/RoomServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.example.demo.service.room;
+
+import com.example.demo.dao.RoomRepository;
+import com.example.demo.entity.Room;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+public class RoomServiceImpl implements RoomService {
```

Comment:
`Impl` to raczej słaba praktyka jeśli chodzi o nazywanie klas. Gdyby nie wydzielać niepotrzebnego interfejsu to nie byłoby problemu z nazwaniem tej klasy ;) Czasami robi się osobną implementację do testów albo dla Repository można zrobić uproszczoną implementację `InMemory` oprócz `Database`. Tutaj z tego co widzę nie ma takiej sytuacji. :)

Reply:
To samo co wyżej.